### ISSUE TITLE
Add Hapi 16 support

### ISSUE THREAD

**Issue by h4m3d** [hapijs/glue#82]

**Comment by WesTyler** [comment]
I have a local commit for this, waiting on approval/merging of #84 before pushing/PRing.

**Comment by AdriVanHoudt** [comment]
@WesTyler You can open the PR for hapi 16 already mentioning the dependency on #84 ;)

**Comment by WesTyler** [comment]
@AdriVanHoudt the tests will fail on CI checks if I push them separately. The Joi 10.x upgrade requires #84 to be in place or else an error is thrown.

Is it kosher for me to push both bits into the same PR so that the tests will pass? I know they are both small changes, but I like to keep PRs as focused as possible.

**Comment by AdriVanHoudt** [comment]
@WesTyler there is no harm in doing a PR that makes CI fail, we even encourage people to submit PR's with failing tests to 'prove' there is a bug which makes it way easier to fix!

And I do think separate PR's are fine here ^^

Thanks for the contribution!!
